taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs

```diff
@@ -177,15 +177,15 @@ private void IfConditionHelper_CellContentClick(object sender, DataGridViewCellE
 
                     var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginIfCommand).ToList();
```

Comment:
Could likely be reduced to CommandControls.GenerateCommandsandControls

Reply:
Same is the case here that it's just updating variable names/references in this Command File.